server/runtime.go

```diff
@@ -77,28 +77,22 @@ func (s *Server) CreatePodSandbox(ctx context.Context, req *pb.CreatePodSandboxR
 	dnsOpts := req.GetConfig().GetDnsOptions()
 	fmt.Println(dnsOpts)
 
-	// TODO: the unit of cpu here is cores. How to map it into specs.Spec.Linux.Resouces.CPU?
-	cpu := req.GetConfig().GetResources().GetCpu()
-	if cpu != nil {
-		limits := cpu.GetLimits()
-		requests := cpu.GetRequests()
-		fmt.Println(limits)
-		fmt.Println(requests)
-	}
-
-	memory := req.GetConfig().GetResources().GetMemory()
-	if memory != nil {
-		// limits sets specs.Spec.Linux.Resouces.Memory.Limit
-		limits := memory.GetLimits()
-		if limits != 0 {
-			g.SetLinuxResourcesMemoryLimit(uint64(limits))
-		}
+	// set up CPU resources
+	limits := req.GetConfig().GetResources().GetCpu().GetLimits()
```

Comment:
@runcom , I had the same doubt at first.
I check the `api.pb.go`, and it already took care the cases where the receiver is `nil`. Like this:

    if m != nil {
        return m.Memory
    }
    return nil